pkg/hubble/metrics/api/context.go

```diff
@@ -201,6 +215,26 @@ func ParseContextOptions(options Options) (*ContextOptions, error) {
 			if err != nil {
 				return nil, err
 			}
+		case "destinationegresscontext":
```

Comment:
I think we should validate that these new options are not set with their mutually exclusive options.

Reply:
@chancez They are mutually exclusive only if both XxxIngressContext and XxxEgressContext are set. Setting XxxIngressContext and XxxContext together is a valid setting.

Even when all three XxxIngressContext, XxxEgressContext and XxxContext are set it theoretically can be a valid setting if we have a flow with direction UNKNOWN.

Reply:
Ah right. Good point.